test/types/mongo.test-d.ts

```diff
@@ -38,6 +38,34 @@ expectAssignable<Model>({
   list: [3, 3, 3]
 });
 
+expectAssignable<UpdateManyInput<Model["list"][0]>>({
```

Comment:
```suggestion
expectAssignable<UpdateManyInput<Model["list"][number]>>({
```
Please change on other places too

Reply:
Done